### ISSUE TITLE
DEBUG numba

### ISSUE THREAD

**Issue by Ayushjaiswal12** [jaywalnut310/vits#66]
I am having some debug numba core  output that is running infintely. Please help me build this model on my own dataset.

**Comment by kradonneoh** [comment]
At least for the numba debug issue, I assume you're running this in the Google Colab notebook. It's not running infinitely, the stdout on the colab is just so slow that it halts the import process.

The import that causes this issue in `librosa`, which has [documented issues](https://stackoverflow.com/questions/66947758/load-audio-from-google-storage-using-google-colab-python) with Colab. The way I solved this was by importing `librosa` before the code in this repo i.e. changing the cell which imports everything to:
```
%matplotlib inline
import matplotlib.pyplot as plt
import IPython.display as ipd

import os
import json
import math
import torch
from torch import nn
from torch.nn import functional as F
from torch.utils.data import DataLoader

import librosa
import logging

logging.getLogger('numba').setLevel(logging.WARNING)

import commons
import utils
from data_utils import TextAudioLoader, TextAudioCollate, TextAudioSpeakerLoader, TextAudioSpeakerCollate
from models import SynthesizerTrn
from text.symbols import symbols
from text import text_to_sequence

from scipy.io.wavfile import write


def get_text(text, hps):
    text_norm = text_to_sequence(text, hps.data.text_cleaners)
    if hps.data.add_blank:
        text_norm = commons.intersperse(text_norm, 0)
    text_norm = torch.LongTensor(text_norm)
    return text_norm
```

The important change is 
```
import librosa
import logging

logging.getLogger('numba').setLevel(logging.WARNING)
```
which stops `numba` from outputting the debug calls, and therefore preventing the blocking stdout messages -- hope this helps!

**Comment by JohnHerry** [comment]
The same problem.  I am using docker of CentOS7, conda, python3.8, CUDA11.7,  torch1.13.1+cu117,
the training process continuelly output the numba debug info, and generate coredump files but training process did not quit.
occasionally, there is a line of report:

'RuntimeError: DataLoader worker is killed by signal: Aborted"

**Comment by ZodiacFRA** [comment]
yep same, any solutions?

**Comment by JohnHerry** [comment]
No, I do not know what happened inside. you can just make clear that there is no bug in your dataloader, and you'd better make pre-processing to filter out bad samples from your training dataset.
